config/jobs/kubernetes-sigs/image-builder/image-builder-periodics.yaml

```diff
@@ -0,0 +1,31 @@
+periodics:
+- name: periodic-image-builder-gcp-all
+  interval: 12h
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: image-builder
+    base_ref: master
+    path_alias: "sigs.k8s.io/image-builder"
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210426-51fd28e-master
+        args:
+          - runner.sh
+          - "./images/capi/scripts/ci-gce.sh"
+        env:
+          - name: BOSKOS_HOST
+            value: "boskos.test-pods.svc.cluster.local"
```

Comment:
Using boskos will mean that the image will be uploaded to random projects that boskos picks up. So no, let's not do this please. we want to push the image specifically to a project (like `"k8s-staging-cloud-provider-gcp"`)

Reply:
ok, thanks for the feedback.
I was thinking a bit more and I will create another script for the nightly and will build images for 1.18/1.19/1.20/1.21 the default case only builds for 1.18.

and one question: when a new job run to we want to delete the image created a day before? or overwrite that (I don't think is possible to overwrite, but then delete and recreate them)

Reply:
made the changes here and in the PR https://github.com/kubernetes-sigs/image-builder/pull/445 to have a dedicated script

Reply:
thanks!